source/_docs/installation/hassbian.markdown

```diff
@@ -17,3 +17,10 @@ Hassbian is our customized operating system for the Raspberry Pi Zero, 2,3 and 3
  - [Pi specific integrations](/docs/hassbian/common-tasks/)
  - [Learn how to perform common tasks](/docs/hassbian/integrations/)
 
+### {% linkable_title Activating the virtual environment %}
+
+When instructions tell you to activate the virtual environment to install a Python library, or perform some tasks, you will use the following commands:
```

Comment:
Same as the other PR.

`you will use the following commands` -> Maybe a little bit to commanding...?